### ISSUE TITLE
Supress the build warning

### ISSUE THREAD

**Issue by ryanlin1986** [antlr/antlrcs#37]
I got this in Visual Studio:

```
Warning	4	'Willinsoft.RAD.BPL.IntellisenseDebugGrammarParser.StatementContext.Stop' hides inherited member 'Antlr4.Runtime.ParserRuleContext.Stop'. Use the new keyword if hiding was intended.	E:\Project\Willinsoft Development Platform\Willinsoft.RAD\obj\Debug\IntellisenseDebugGrammarParser.cs	4346	17	Willinsoft.RAD
```

It's better to suppress this warning which makes warning list huge.

**Comment by sharwell** [comment]
Hi @ryanlin1986,

Thanks for taking the time to report this issue. :smile: 

This is actually the repository for the C# ports of ANTLR 3 and StringTemplate. If you are using the ANTLR 4 C# target available on NuGet, the repository you are looking for [is here](https://github.com/tunnelvisionlabs/antlr4cs). Otherwise, if you are using the ANTLR 4 C# target which is included in the ANTLR 4.5 release from antlr.org, the repository you are looking for [is here](https://github.com/antlr/antlr4-csharp).

That said, it sounds like you have a lexer rule named `Stop`, and the methods producing this warning are the generated `Stop()` methods which provide access to these tokens. Is this correct?

**Comment by ryanlin1986** [comment]
Thanks for the response, what you said is partually right.
After checked it out, I found some parser rule assigned by name START and STOP, which produces this warning, I will just rename the name.
